### PR TITLE
textwrangler: manage version for older OSX

### LINKED ISSUE #17482: TextWrangler 5+ requires OSX 10.8.5, breaking cask on older OSX version
OSX Snow Leopard and Lion can use TextWrangler 4.5, as per the [vendor's download page](http://www.barebones.com/support/textwrangler/updates.html), but the cask doesn't handle this.

### PR DESCRIPTION
**Pull request by psibre**
fixes #17482

### CODE REVIEW

**Comment by psibre** [comment]
@vitorgalvao Unlike #17483, this allows me to install TW on my Snow Leopard MacBook without problems.

**Comment by adidalal** [comment]
```
'textwrangler' definition is invalid: 'depends_on :macos' comparison expressions cannot be combined (Hbc::CaskInvalidError)
```

**Comment by psibre** [comment]
@adidalal I saw that in the CI log, but that didn't seem to stop @vitorgalvao from merging #17483, which failed the CI build with the same error.

I guess there is a systematic issue with the `depends_on` stanza, even though it is still used in numerous casks?

**Comment by psibre** [comment]
@vitorgalvao Incidentally, would you like me to rebase this on your merged commit?

File: Casks/textwrangler.rb [hunk 1]
```diff
@@ -1,16 +1,22 @@
 cask 'textwrangler' do
-  version '5.0.2'
-  sha256 '29484008bc95aee2e2731781b514cd60ca1e19fdb5a4b575ac5dbbf79472c346'
+  if MacOS.release <= :lion
+    version '4.5.12'
+    sha256 'f25ce90a54570ee4fe2287cdbe5c8b2a57188423a3693becacfa0533f02e0aa2'
+    url "http://pine.barebones.com/files/TextWrangler_#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/textwrangler.rb]: Would you mind adding an extra empty line above `url`, for consistency?

Comment (psibre) [inline Casks/textwrangler.rb]: sure

**Comment by vitorgalvao** [comment]
> I saw that in the CI log, but that didn't seem to stop @vitorgalvao from merging #17483, which failed the CI build with the same error.

Why do you think that? That Travis build was interrupted, it did not error out with the same error. Even so, if I do a mistake, don’t replicate it, feel free to correct me.

Everything looks good, now, thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by psibre** [comment]
squashed

**Comment by psibre** [comment]
@vitorgalvao 
> Even so, if I do a mistake, don’t replicate it, feel free to correct me.

Sorry, wasn't accusing anybody -- I just had the suspicion that the CI build was failing due to some outdated definitions, of which both you *and* I had independently run afoul. But now I see the builds failed for different reasons.

**Comment by vitorgalvao** [comment]
> Sorry, wasn't accusing anybody

Oh, that wasn’t my reaction (apologies if it seemed like that, I did try to not make it sound like it, but struggled a bit since we’re communicating in writing). I did really mean it in the sense of “it’s very possible that I’ve done a mistake there, so if I did trust the tests more than you trust me, and do it the right way”. All in good terms.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 68c92270de792f06bc564be7263241fc4eebae63 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by psibre** [comment]
@vitorgalvao Thanks, no worries! And sorry to you (and Travis) for the wasted cycles. =)

**Pull request closed without merging**